math/doc.go

```diff
@@ -26,12 +25,7 @@ outline: math
     atan(x)
       Return the arc tangent of x, in radians.
     atan2(y, x)
-      Return atan(y / x), in radians. The result is between -pi and pi.
-      The vector in the plane from the origin to point (x, y) makes this angle
-      with the positive X axis. The point of atan2() is that the signs of both
-      inputs are known to it, so it can compute the correct quadrant for the
-      angle. For example, atan(1) and atan2(1, 1) are both pi/4, but
-      atan2(-1, -1) is -3*pi/4.
+      Return atan(y / x), in radians. The result is between -pi and pi. The vector in the plane from the origin to point (x, y) makes this angle with the positive X axis. The point of atan2() is that the signs of both inputs are known to it, so it can compute the correct quadrant for the angle. For example, atan(1) and atan2(1, 1) are both pi/4, but atan2(-1, -1) is -3*pi/4.
```

Comment:
I know this was taken from the python docs themselves but I'm mad at the writer because why would you use the phrase "the point of" when defining a function that deals with the cartesian coordinate system????

"The point of atan2() is that the signs of both inputs are known to it, so it can compute the correct quadrant for the angle." => "The atan2() function can compute the correct quadrant for the angle since it knows the sign of both inputs."

Is how I would personally re-phrase this.